common/env.ts

```diff
@@ -20,20 +20,72 @@ export const DD_SERVICE_ENV_VAR = "DD_SERVICE";
 export const DD_VERSION_ENV_VAR = "DD_VERSION";
 export const DD_TAGS = "DD_TAGS";
 
-export function setGitCommitEnvironmentVariables(lambdas: any[], hash: string, gitRepoUrl: string | undefined) {
+const execSync = require("child_process").execSync;
+
+const URL = require("url").URL;
+
+export function setGitEnvironmentVariables(lambdas: any[]) {
+  log.debug("Adding source code integration...");
+  const [hash, gitRepoUrl] = getGitData();
```

Comment:
nit: I'm usually against returning lists because it doesn't tell us anything about the meaning of the return values. Might be better to return an object, but it probably doesn't matter for this use case.

Reply:
addressed